lib/actions/device.coffee

```diff
@@ -180,6 +180,21 @@ exports.remove =
 			resin.models.device.remove(params.uuid)
 		.nodeify(done)
 
+exports.ping =
+    signature: 'device ping <uuid>'
+    description: 'ping a device with a UUID'
+    help: '''
+           Use this command to ping a device.
+
+           Examples:
+
+                   $ resin device ping 23c73a1
+    '''
+    permission: 'user'
+    action: (params, options, done) ->
+        resin = require('resin-sdk-preconfigured')
```

Comment:
These errors are because all of your code is space-indented I think - this should all be tabbed.